### DISCUSSION TITLE
crashing after every song

### DISCUSSION THREAD

**Discussion by sigmavan** [KadeArchive/Kade-Engine#1381] [Q&A]
my game crashes whenever i finish a song. this happens in story mode and freeplay. this isnt just with hex, every kade engine mod i download has this issue. with the last mod i tried on kade engine (sky mod) i reinstalled twice, issue remained. 
here is the error message, no idea what it means.
i thought it would be an issue with my antivirus (bitdefender), however this wasnt the case as i disabled it and played through again and the same issue happened.
![image](https://user-images.githubusercontent.com/78553509/126224276-1b1f2529-8d0b-4726-b0a5-9b490b2459ae.png)

**Comment by ghost** [comment]
i think it might be it trys to save the kadereplay but the directory dosent exitst if not idk

**Comment by KlutchDj** [comment]
uhmmm , pretty sure this is on your end. you should have a replay folder in your assets directory , if not, thats the reason why it crashes because its not finding its correct path.

**Comment by KlutchDj** [comment]
or something idk , could just be me , but its just a assumption
